Java/SearchRepeatingElements.java

```diff
@@ -0,0 +1,16 @@
+package searching;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class SearchRepeatingElements {
+    public static void main(String args[]){
+        int arr[]={1,2,2,5,7,4,5,8,7};
+        Set<Integer> set = new HashSet<>();
+        for(int i=0;i<arr.length;i++){
+            if(!set.add(arr[i])){
+                System.out.print(arr[i]+" ");
+            }
+        }
+    }
+}
```

Comment:
Add Sample I/P, O/P, Time and Space complexity in a single MULTILINE COMMENT.
Let me know once you're done @garv-cmd

Reply:
Done!!